velox/type/DecimalUtils.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <string>
+#include "velox/common/base/Exceptions.h"
+#include "velox/type/LongDecimal.h"
+#include "velox/type/ShortDecimal.h"
+#include "velox/type/Type.h"
+
+namespace facebook::velox {
+
+template <typename T>
+std::string decimalToString(const T& value, const TypePtr& type) {
```

Comment:
BTW, using this function as a building block for cast operator will require expensive string copying from std::string to string buffer of the vector. We'd need a different design to avoid that copy.

Reply:
@mbasmanova lets use this only for debugging purposes. We will write a separate implementation for cast.

Reply:
Sounds good. Let's document this function though.